Add table aws_cloudtrail_channel closes #1504 (#1517)
turbot/steampipe-plugin-aws: docs/tables/aws_cloudtrail_channel.md

> # Table: aws_cloudtrail_channel
> AWS services can create a service-linked channel to receive CloudTrail events on your behalf. The AWS service creating the service-linked channel configures advanced event selectors for the channel and specifies whether the channel applies to all regions, or a single region.

Reviewer: ```suggestion
# Table: aws_cloudtrail_channel

AWS services can create a service-linked channel to receive CloudTrail events on your behalf. The AWS service creating the service-linked channel configures advanced event selectors for the channel and specifies whether the channel applies to all regions, or a single region.
```

## Examples

### Basic info

```sql
select
  name,
  arn,
  source,
  apply_to_all_regions
from
  aws_cloudtrail_channel;
```

> ### List channels that are not applyed to all region

Reviewer: ```suggestion
### List channels that are not applied to all regions
```

```sql
select
  name,
  arn,
  source,
  apply_to_all_regions,
  advanced_event_selectors
from
  aws_cloudtrail_channel
where
  not apply_to_all_regions;
```

> ### Get advance event selector details for each channel

Reviewer: ```suggestion
### Get advance event selector details of each channel
```
